qa/TL0_framework_imports/test_mxnet.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+# used pip packages
+
+pip_packages="mxnet-cu{cuda_v}"
```

Comment:
you don't need other packages in those? numpy? Just wondering

Reply:
No. The whole test code is just:
```
python -c "import mxnet; import nvidia.dali.plugin.mxnet"
```